vpc.tf

```
@@ -1,51 +1,42 @@
+locals {
+  public_cidr = ["10.0.0.0/24","10.0.1.0/24","10.0.2.0/24"]
+  private_cidr = ["10.0.100.0/24","10.0.101.0/24","10.0.103.0/24"]
+  availability_zones = ["us-east-1a", "us-east-1b","us-east-1c"]
```

```suggestion
  public_cidr        = ["10.0.0.0/24","10.0.1.0/24","10.0.2.0/24"]
  private_cidr       = ["10.0.100.0/24","10.0.101.0/24","10.0.103.0/24"]
  availability_zones = ["us-east-1a", "us-east-1b","us-east-1c"]
```

equal signs must be aligned, you can also run terraform fmt to have this resolved automatically